<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>793</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Miner's Lamp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/eth_miner_lamp.ico</normaloff>:/icons/eth_miner_lamp.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {

color: white;
  background-color:  black;

 }

QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #00b400, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}

QLabel
{
  background-color: black;
  color: white;

}

QGroupBox {
    background-color: black;
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: black;
}


QLineEdit {
 padding: 1px;
 border-style: solid;
 border: 2px solid gray;
 border-radius: 4px;
 }

QTextEdit {
  border: 2px solid gray;
  border-radius: 4px;
}


QLineEdit#lineEditDonate
{
border-style: no-border;
color: yellow;
}


QDateTimeEdit {
  padding: 1px;
  border-style: solid;
  border: 2px solid gray;
  border-radius: 4px;
}

QTimeEdit {
  padding: 1px;
  border-style: solid;
  border: 2px solid gray;
  border-radius: 4px;
}

QComboBox {
  border: 2px solid gray;
  border-radius: 4px;
}

QTreeView {
  border: 2px solid gray;
  border-radius: 4px;
}


QListView {
    alternate-background-color: gray;
}



QCheckBox {
color: white;

 }

QSpinBox {
color: white;
 padding: 1px;
 border-style: solid;
 border: 2px solid gray;
 border-radius: 2px;
 }

QPushButton {
 color: white;
 background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #808080, stop: 0.1 #909090,
                    stop: 0.49 #505050, stop: 0.5 #666666, stop: 1 #909090);
 border-width: 1px;
 border-color: black;
 border-style: solid;
 border-radius: 4;
 padding: 3px;
 font-size: 13px;
 padding-left: 5px;
 padding-right: 5px;
 min-width: 50px;
 max-width: 200px;
 min-height: 13px;
 max-height: 40px;
 }

QPushButton:pressed
{
 color: white;
 background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #77e, stop: 0.49 #12c, stop: 0.5 #33b, stop: 1 #22c);
 border-width: 1px;
 border-color: black;
 border-style: solid;
 border-radius: 4;
 padding: 3px;
 font-size: 13px;
 padding-left: 5px;
 padding-right: 5px;
 min-width: 50px;
 max-width: 200px;
 min-height: 13px;
 max-height: 40px;
}

QPushButton:hover
{
 color: white;
 background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #505050, stop: 0.1 #707070, stop: 0.49 #12c, stop: 0.5 #aaaaaa, stop: 1 #33c);
 border-width: 1px;
 border-color: black;
 border-style: solid;
 border-radius: 4;
 padding: 3px;
 font-size: 13px;
 padding-left: 5px;
 padding-right: 5px;
 min-width: 50px;
 max-width: 200px;
 min-height: 13px;
 max-height: 40px;
}

QPushButton:checked
{
 color: white;
 background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #77e, stop: 0.49 #12c, stop: 0.5 #44b, stop: 1 #ddc);
 border-width: 1px;
 border-color: black;
 border-style: solid;
 border-radius: 4;
 padding: 3px;
 font-size: 13px;
 padding-left: 5px;
 padding-right: 5px;
 min-width: 50px;
 max-width: 200px;
 min-height: 13px;
 max-height: 40px;
}

QPushButton:disabled
{
 color: white;
 background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
                    stop: 0 #CCCCCC, stop: 0.1 #EEEEEE,
                    stop: 0.49 #AAAAAA, stop: 0.5 #BBBBBB, stop: 1 #CCCCCC);
 border-width: 1px;
 border-color: black;
 border-style: solid;
 border-radius: 4;
 padding: 3px;
 font-size: 13px;
 padding-left: 5px;
 padding-right: 5px;
 min-width: 50px;
 max-width: 200px;
 min-height: 13px;
 max-height: 40px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="5" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBoxAMD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>AMD</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="11">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MinPoxer"/>
       </item>
       <item row="2" column="5">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MinFan"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Max Speed fan:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MaxTemp"/>
       </item>
       <item row="1" column="5">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MaxFan"/>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Max mem clock:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Min mem clock:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="9">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MinMemClock"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Min GPU temp:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MinClock"/>
       </item>
       <item row="1" column="9">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MaxMemClock"/>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Min GPU clock:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_AMD_GPUCount">
         <property name="text">
          <string>Devices number:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Max GPU clock:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MinTemp"/>
       </item>
       <item row="1" column="11">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MaxPower"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Max GPU temp:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Min fan speed:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="10">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Min power draw:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLCDNumber" name="lcdNumber_AMD_MaxClock"/>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Max power draw:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="lcdNumber_AMD_GPUCount">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBoxPools">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Pools</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Average Hashrate
for last 6 hour:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Current Calculated Hashrate:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QPushButton" name="pushButtonDisplayPoolStats">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLCDNumber" name="lcdNumberCalculatedHR">
         <property name="digitCount">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Last Reported Hashrate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonNaopool">
         <property name="text">
          <string>Nanopool</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="radioButtonDwarfPool">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Dwarfpool</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QLineEdit" name="lineEditAccount">
         <property name="text">
          <string>0xa07A8c9975145BB5371e8b3C31ACb62ad9d0698E</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButtonEthermine">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ethermine</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLCDNumber" name="lcdNumberBalance">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLCDNumber" name="lcdNumber_4">
         <property name="digitCount">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Balance:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLCDNumber" name="lcdNumberAvrgHr6H">
         <property name="digitCount">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBoxNvidia">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>nVidia</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="10">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Max mem clock:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Devices number:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Min fan speed:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumberGPUCount">
         <property name="digitCount">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QLCDNumber" name="lcdNumberMaxMemClock"/>
       </item>
       <item row="0" column="7">
        <widget class="QLCDNumber" name="lcdNumberMaxFanSpeed"/>
       </item>
       <item row="0" column="13">
        <widget class="QLCDNumber" name="lcdNumberMaxWatt"/>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Max Speed fan:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Max GPU temp:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLCDNumber" name="lcdNumberMaxGPUTemp">
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLCDNumber" name="lcdNumberMinGPUTemp">
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QLCDNumber" name="lcdNumberMinMemClock"/>
       </item>
       <item row="0" column="12">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Max power draw:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Min GPU temp:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Min mem clock:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLCDNumber" name="lcdNumberMinFanSpeed"/>
       </item>
       <item row="1" column="12">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Min power draw:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="13">
        <widget class="QLCDNumber" name="lcdNumberMinWatt"/>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QPushButton" name="pushButtonOC">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>OverClocking</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Min GPU clock:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Max GPU clock:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QLCDNumber" name="lcdNumberMaxGPUClock"/>
       </item>
       <item row="1" column="9">
        <widget class="QLCDNumber" name="lcdNumberMinGPUClock"/>
       </item>
       <item row="2" column="12">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Total power draw:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="13">
        <widget class="QLCDNumber" name="lcdNumberTotalPowerDraw">
         <property name="digitCount">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QGroupBox" name="groupBoxWatchdog">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Watchdog options (restart miner in case of error)</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Max consecutive 0MH/s:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBoxMax0MHs">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Delay before restart:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="spinBoxDelay">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Delay before monitoring for 0MH/s:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxDelay0MHs">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Delay before no hash:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBoxDelayNoHash">
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEditArgs"/>
    </item>
    <item row="8" column="0" colspan="6">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="2">
     <widget class="QLCDNumber" name="lcdNumberHashRate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButtonShowHideLog">
      <property name="text">
       <string>Hide log</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="pushButtonPool">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Pool</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Command line:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButtonEthminerBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxAutoStart">
      <property name="text">
       <string>Auto start miner</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButtonHelp">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Miner path:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxOnlyShare">
      <property name="text">
       <string>Only display share info</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start mining</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditMinerPath"/>
    </item>
    <item row="6" column="0" colspan="6">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Hashrate chart</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="hashrateCharView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>hashrateCharView</class>
   <extends>QGraphicsView</extends>
   <header>hashratecharview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
